packages/connect/src/index.d.ts

```diff
@@ -45,7 +45,7 @@ type ResolveActions<Act extends Actions> = {
     ? () => ReturnType<Act[P]>
     : Act[P] extends Actions
     ? ResolveActions<Act[P]>
-    : never;
+    : any;
```

Comment:
What does this line fix? There is nothing related to `connect` in the PR description. :thinking:

Reply:
It fixes another TypeScript problem, related to these lines https://github.com/frontity/frontity/blob/dev/packages/core/src/server/index.tsx#L95. It was complaining that `never` was not callable.

I'll try to improve the TypeScript coverage of that part once I reorganize our types. Those types are a bit of a mess right now.